distributed/tests/test_core.py

```diff
@@ -150,29 +152,58 @@ def listen_on(cls, *args):
             yield assert_can_connect(server.address)
             yield assert_can_connect_locally_6(server.port)
 
+    # InProc
 
-def test_rpc(loop):
-    @gen.coroutine
-    def f():
-        server = Server({'ping': pingpong})
-        server.listen(8883)
+    with listen_on(Server, 'inproc://') as server:
```

Comment:
That was meant to verify that the basic `core` infrastructure works with inproc. I think a "grep inproc" would easily point the places where it's used :-)